### DISCUSSION TITLE
Trabalhar remoto em casa de praia de cidade pequena ?

### DISCUSSION THREAD

**Discussion by romulosardinha** [frontendbr/forum#1497] [Discussão]
E ai? quero a opinião de vocês. Trocariam a cidade grande por uma cidade pequena? Onde pela metade do valor do aluguel você estaria em uma casa de praia? Melhor qualidade de vida, menos gastos. E abandonar os itens de cidade grande ?

**Comment by bernardodiasc** [comment]
sim, com certeza haha

sou suspeito pra falar, trabalho remoto há mais de uma década, nunca dependi de renda do locais que vivi e isso me permitiu escolher onde morar. 

mas vai de gosto, tem que goste de praia, tem quem goste de montanha, tem quem goste de fazenda e tem que goste de cidades. o mais interessante mesmo não é a escolha mas sim poder escolher. :)

**Comment by romulosardinha** [comment]
kk pois é, falei de praia porque é a oportunidade que me apresenta agora. Tbm sou remoto a uns 7 anos. Mas está dando uma dor no coração não ir mais ao cinema kkk.

**Comment by bernardodiasc** [comment]
Mano... vai ver as ondas chegarem na praia, mil vezes melhor que qualquer filme kkkkkkkkk 

Ja que é remoto a tanto tempo deve ter alcançado uma certa estabilidade né? Creio que nada te impeça de experimentar as possibilidades. Fica um tempo no lugar e vê se as coisas que tinha antes realmente fazem falta ou se voce as usava somente por te-las como opção. Lembrando que em lugares diferentes as opções também são diferentes, pode ser realmente que nem se lembre que cinema existe (meu caso). hehehe

**Comment by romulosardinha** [comment]
Esquecer cinema ???? Difícil. kk mais fácil eu comprar um projetor para fingir que estou no cinema kk

**Comment by romulosardinha** [comment]
Mas é exatamente, No lugar de ir ao shopping, vou aprender a surfar. As opções só mudaram, concordo contigo.

**Comment by kivervinicius** [comment]
Qual cidade? dependendo do estado compensa andar uns km no fds para ver gente hehe, meu sonho ter uma proposta assim! kk

**Comment by romulosardinha** [comment]
@kivervinicius ainda não tenho certeza da cidade, estou olhando algumas ainda, mas por exemplo, tem Guriri ES, Luís Correia PI, cidades pequenas praieiras, que estão com criminalidade quase zero, qualidade de vida, ótimo custo...

**Comment by kazzkiq** [comment]
Aceitaria sem pensar duas vezes!

Meu sonho é morar em uma cidade pequena cercada de natureza. As vezes até fico pesquisando valor de aluguel em cidades no interior de MG, mesmo sem ter condições de ir pra lá. Só de pesquisar já dá uma tranquilidade... imagina se fosse numa cidade litorânea!

Se você tem estabilidade no trampo remoto pra dar essa arriscada, eu diria pra aproveitar sim! No pior dos mundos, seria uma experiência muito louca morar um tempo na praia.

----

Edit porque esqueci de falar disso: É claro que também é importante escolher uma cidade com internet boa né, já tivemos experiências aqui na empresa de gente trabalhando remota e tendo diversos problemas de comunicação porque o provedor de internet da região dava problema todo mês.

**Comment by kivervinicius** [comment]
@rmlSardinha já pensou em Bertioga? Internet é top, e se precisar de algo SP ta a poucos km, não conheço essas cidades q mencionou, mas vou olhar, estou só esperando minha esposa formar para arriscar algo do tipo.

**Comment by romulosardinha** [comment]
@kivervinicius, pelo que estou vendo o custo de vida é alto ai. Mano, tem cidadezinha que casa de uns 300 m² de frente para praia está R$ 700,00 por mês.

**Comment by kivervinicius** [comment]
ai é top, qualquer coisa avisa ai fiquei interessado hehe

**Comment by felipeuntill** [comment]
Meu sonho, comprei um terreno em um condomínio em santa isabel para isso, entretanto no meu caso oque impera é a coragem de pedir para a mulher deixar de trabalhar e fazer o relocation comigo.

**Comment by romulosardinha** [comment]
@kivervinicius pode deixar, quando eu mudar vou postar tudo por aqui. custos da mudança, internet, etc

**Comment by ViniciusGularte** [comment]
Sonho hehe, ou até ir pra outro país com imposto mais em conta

**Comment by fernandoRj** [comment]
se eu arrumar um job fixo remoto, nao penso 2x antes de ir para tailandia

**Comment by romulosardinha** [comment]
@fernandoRj nem esperei o job fixo kk to indo e só pego freelas remotos

**Comment by ypedroo** [comment]
Nenhuma das tuas possibilidades envolve cidades próximas de grandes centros? Aqui no CE por exemplo do ladinho de Fortaleza temos uma cidade chamada Aquiraz e em uns 30 minutinhos o cara ta nos polos que te interessam e tal.

**Comment by kivervinicius** [comment]
@fernandoRj pq tailandia?

**Comment by romulosardinha** [comment]
@ypedroo opa, não conhecia essa... está barata ?

**Comment by bbertolani** [comment]
@ypedroo gostei dessa Aquiraz! Valeu pela dica!

**Comment by ypedroo** [comment]
Cara é quase na média de Fortaleza, mas Fortaleza é uma das capitais mais baratas do Br.

**Comment by bbertolani** [comment]
@ypedroo casa com piscina, 5 quartos 1200 conto! eu to achando muito barato! To no interior de SP e nao acho nada assim

**Comment by romulosardinha** [comment]
@bbertolani viu kkk isso ai.

**Comment by romulosardinha** [comment]
@ypedroo só que fortaleza é pior que o rio em questão de crime.

**Comment by ypedroo** [comment]
Brother tendo vivido em ambas pior não é mas não, mas também não ta muito atras, é bom verificar os indices destas cidades próximas. Normalmente é menor

**Comment by romulosardinha** [comment]
@ypedroo https://g1.globo.com/ce/ceara/noticia/fortaleza-e-a-setima-cidade-mais-violenta-do-mundo-diz-ong-do-mexico.ghtml sei lá. Não gosto dos números dessa capital

**Comment by jair-anderson-souza** [comment]
Deem uma olhada em João Pessoa, é uma capital relativamente pequena, aluguel nem é tão caro assim, acho que compensa vir morar por aqui.

**Comment by brunosantanarj** [comment]
Eu te sugiro conhecer as cidades vizinhas, analisar quanto tempo gasta para trafegar de uma cidade a outra para ir ao cinema, por exemplo.

Eu penso da seguinte maneira: Quantas vezes na semana vou fazer tal coisa ? 

Por exemplo: Tenho costume de ir ao cinema 1x por semana. Uma cidade sem cinema poderia me desanimar após alguns meses, pois eu já tinha o costume desse entretenimento. 

Algumas perguntas que eu faria para mim antes de tomar essa decisão: 

- Eu amo praia e sempre o tive costume de ir ?

- Nessa cidade(ou próximas) tem as coisas mínimas aceitáveis que eu curto fazer ou vou levar uma hora de estrada ?

Eu passei por isso algumas semanas atrás, e após pensar bastante, vi que não valeria(Quando estava num sábado a noite na cidade e não achei um local maneiro para comer) a pena no momento. Por ser casado e sem filhos, costumo ter uma vida mais noturna, ir a bares e restaurantes, fazer programas de casal e etc.

Talvez com filhos e família grande para mim faria mais sentido um local teoricamente maior, mais barato, calmo e seguro.

**Comment by eliseumds** [comment]
A tonelada de devs nas praias da Tailândia e Indonésia diriam que sim :P
